test/ui_test_label_studio.py

```diff
@@ -0,0 +1,213 @@
+        self.driver.switch_to.frame(iframe_)
+        self.driver.find_element_by_xpath(self.Setup_label_car)
+        time.sleep(5)
+        #
+        #Doing 3 clicks for create area
+        action = webdriver.common.action_chains.ActionChains(self.driver)
+        action.move_to_element_with_offset(self.driver.find_element_by_tag_name('body'), 0,0)
+        #
+        # el=self.driver.find_element_by_xpath('/html/body/div/div/div/div/div/div/div[1]/div/div/div[1]/div[2]/div/canvas')
+        # action = webdriver.common.action_chains.ActionChains(self.driver)
+        # action.move_to_element_with_offset(el, 500, 500)
+        # action.click()
+        # action.perform()
+        #
+        x = 700
+        y = 300
+        action.move_by_offset(x, y).click().perform()
+        action.move_by_offset(x, y+50).click().perform()
+        action.move_by_offset(x+30, y+30).click().perform()
+        action.move_by_offset(x, y).click().perform()
+        #
+        #Go to outsite from iframe
+        self.driver.switch_to_default_content()
+
+
+    #Step2 - Import tasks to project
+    def step2(self, testCaseName):
+        # Go to page Import
+        print("..... Going to page Import .....")
+        self.driver.find_element_by_xpath(self.welcome_Import).click()
```

Comment:
I think it's better to extract checks on the page into distinct step/method and separate them from the way you get on this page, because this can be explicit like here (direct link) or implicit (like after submit and redirect).

Reply:
and this can be a part of test — to check if the browser redirected automatically to another page after some action

Reply:
I think we will add another test chain for that)